documentation/src/docs/asciidoc/user-guide/junit-platform-suite-engine.adoc

```diff
@@ -0,0 +1,41 @@
+[[junit-platform-suite-engine]]
+=== JUnit Platform Suite Engine
+
+The JUnit Platform supports declarative definition and execution of suites of tests from
+_any_ test engine using the Junit Platform.
+
+==== Setup
+
+In addition to _at least one_ other test engine, you need the following artifacts and
+their dependencies on the classpath. See <<dependency-metadata>> for details regarding
+group IDs, artifact IDs, and versions.
+
+===== Explicit Dependencies
+
+* `junit-platform-suite-api` in _test_ scope
+* `junit-platform-suite-engine` in _test runtime_ scope: implementation of the
+                                `TestEngine` API for the declarative Junit Platform
+                                Suites
+
+===== Transitive Dependencies
+
+* `junit-platform-launcher` in _test_ scope
+* `junit-platform-engine` in _test_ scope
+* `junit-platform-commons` in _test_ scope
+* `opentest4j` in _test_ scope
+
+==== Test Suite
+
+By annotating a class with `@Suite` it is marked as a test suite on the JUnit Platform.
+As seen in the following example, selector and filter annotations can then be used to
+control the contents of the suite.
+
+[source,java,indent=0]
+----
+include::{testDir}/example/SuiteDemo.java[tags=user_guide]
+----
+
+.Additional Configuration Options
```

Comment:
What about meta annotations?

Reply:
In general, JUnit 5 always supports using annotations as meta-annotations. So if you're talking about finding the `@Suite` annotation as a meta-annotation, yes, we should support that.